python_ta/cfg/visitor.py

```diff
@@ -201,7 +201,7 @@ def _visit_jump(self, node: Union[astroid.Break, astroid.Continue, astroid.Retur
         self._current_block = unreachable_block
 
     def visit_tryexcept(self, node: astroid.TryExcept) -> None:
-        self._current_block = self._current_cfg.create_block(self._current_block)
+        self._current_block = self._current_block and self._current_cfg.create_block(self._current_block)
```

Comment:
I'm not a fan of this style. Revert the `__bool__` and just do `if self._current_block.statements != []`.